core/src/main/java/org/elasticsearch/index/mapper/MapperBuilders.java

```diff
@@ -60,6 +61,10 @@ private MapperBuilders() {}
         return new StringFieldMapper.Builder(name);
     }
 
+    public static KeywordFieldMapper.Builder keywordField(String name) {
```

Comment:
Do we actually need this? When I was working on the many refactorings last year, I started removing these, at least for the metadata fields. They seem to only be used by tests afaict.